setup.cfg

```diff
@@ -47,7 +47,7 @@ install_requires =
     packaging
     pluggy>=0.12,<1.0.0a1
     py>=1.8.2
-    toml
+    tomli>=1.0.0,<2.0.0
```

Comment:
I don't think the `>=1.0.0` is needed here, given that there never was a < 1.0.0 release. Not sure about the `<2.0.0` part. We don't typically add upper bounds for dependencies, and I guess the tiny bit of API we need will probably stay the same across future versions.

Reply:
There are [<1.0.0 releases](https://pypi.org/project/tomli/#history). I had this same discussion [with Black maintainers recently](https://github.com/psf/black/pull/2301#discussion_r647748572) :smile: . You'll find my thoughts in that thread.

If you tell me exactly what constraint you want I'll change it to that of course (or you can just push to this branch).

Reply:
Huh, I thought I had looked at PyPI, but apparently I messed up somehow. Ok, seems fine to me either way, let's see what others think.

Reply:
I'm also not inclined to not having the upper pinning in there, because I don't consider `pytest` a user facing application, but a framework, so our dependencies need to be compatible downstream with user's code.

But having said that, I understand the points @hukkin made in that thread, and given he is the main author, his opinion here carries some weight. Also, seems we might never even see a 2.0 version. 😁 

In summary: -0 on the upper pinning.

Reply:
I'll put my -1 on the upper pinning, please remove it

Reply:
Oh sorry, had enough approvals that I thought it would be OK to merge. 👍